examples/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
nonono. don't do that.
examples is not a Python package.

Reply:
:P removed...